src/components/ApplicationMenu/ApplicationMenu.js

```diff
@@ -150,12 +158,24 @@ class ApplicationMenu extends Component<Props> {
     // During onboarding, there is no selected project (because none exists
     // yet). Therefore, we only want to show the 'Project' menu when a project
     // is selected.
-    if (selectedProjectId) {
+    if (selectedProject) {
       // The `Project` menu should be inserted right after `Edit`, which will
       // have a different index depending on the platform.
       const editMenuIndex = template.findIndex(menu => menu.id === 'edit');
 
-      let submenu = [];
+      let submenu = [
+        {
+          label: isMac ? 'Open in Finder' : 'Open in Explorer',
+          click: () => openProjectInFolder(selectedProject),
+          accelerator: 'CmdOrCtrl+shift+F',
+        },
+        {
+          label: isMac ? 'Open in Code Editor' : 'Open in code editor',
+          click: () => openProjectInEditor(selectedProject),
+          accelerator: 'CmdOrCtrl+shift+E',
```

Comment:
I chose these shortcuts somewhat randomly. If y'all have suggestions based on how other apps do it, lemme know!

Reply:
I think that's OK because you added the shift key.
Because Ctrl+F should be reserved for text search if we ever have one.